src/test/java/org/homeschoolpebt/app/cli/TransmitterCommandsTest.java

```diff
@@ -127,15 +134,30 @@ void setup() {
     submissionRepository.save(docUploadOnly);
 
     transmission = Transmission.fromSubmission(docUploadOnly);
-    transmission.setConfirmationNumber("1003");
+    transmission.setConfirmationNumber(String.format("%d", transmissionId));
     transmissionRepository.save(transmission);
+    transmissionId++;
 
     docfile = new UserFile();
     docfile.setFilesize(10.0f);
     docfile.setSubmission_id(docUploadOnly);
     docfile.setOriginalName("laterdoc.png");
     userFileRepository.save(docfile);
 
+    var submissionWithoutSignature = Submission.builder()
+      .submittedAt(now())
+      .flow("pebt")
+      .urlParams(new HashMap<>())
+      .inputData(Map.of(
+        "firstName", "Testing",
+        "lastName", "McOtherson"
+      )).build();
+    submissionRepository.save(submissionWithoutSignature);
+    transmission = Transmission.fromSubmission(submissionWithoutSignature);
+    transmission.setConfirmationNumber(String.format("%d", transmissionId));
+    transmissionRepository.save(transmission);
+    transmissionId++;
```

Comment:
Should there be some assertion here that it isn't being transmitted?

Reply:
I think it's good to add a clear assertion.

When I originally developed this code change, I relied on pre-existing assertion on the number of files in the ZIP. Since it didn't increase, I knew my code was right. However, adding a clear assertion is a good move.